charts/opentelemetry-collector/templates/_pod.tpl

```diff
@@ -17,7 +17,7 @@ containers:
       - /{{ .Values.command.name }}
     {{- end }}
     args:
-      {{- if .Values.configMap.create }}
+      {{- if or .Values.configMap.create .Values.configMap.existingName }}
       - --config=/conf/relay.yaml
```

Comment:
We need to make is clear in the values.yaml file that if you use `existingName` the confmap must contain relay.